binr/rabin2/rabin2.c

```diff
@@ -3,6 +3,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <getopt.c>
+#include <dlfcn.h>
```

Comment:
totally not portable. I wrote r_lib for a reason. use that instead of -ldl. You have the same functionality, but portable and it's shipped for free.